qemu/tests/block_resize.py

```diff
@@ -201,8 +201,11 @@ def get_md5_of_file(filename):
 
             if not utils_disk.is_mount(partition, dst=mpoint,
                                        fstype=fstype, session=session):
-                utils_disk.mount(partition, mpoint,
-                                 fstype=fstype, session=session)
+                session.cmd_output_safe("mkdir -p %s" % mpoint)
```

Comment:
Only create the new mount point when the disk is changed, e.g. sdb1 -> sda1.
Here, if the disk is still sdb1, we don't need to create /mnt/sdb1, which was already present